content/03.methods.md

```diff
@@ -117,7 +117,12 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], for WGS
 
 #### VCF annotation and MAF creation
 
-We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
+Normalization of INDELs using `bcftools norm` [@url:http://samtools.github.io/bcftools/bcftools.html#norm] was performed on all PASS VCFs using the following subworkflow [@url:https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_annot_vcf_sub_wf.cwl], release v3.
+The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, was used to annotate variants and bcftools was used to add population allele frequency (AF) from gnomAD.
+SNV and INDEL hotspots from v2 of MSKCC's database [@url:https://www.cancerhotspots.org/#/download] plus the C228T and C250T TERT promoter mutations [@doi:10.3390/ijms21176034] were annotated.
```

Comment:
>Thanks for these updates! They look like they describe the methods well. One thought was that we should probably specify which analyses use the hotspot file in addition to the main consensus, as not all do. At this point, I think we are using the hotspot file in molecular subtyping analyses, but not in more generalized analyses of mutation rates, etc.

This makes sense, but we probably should add it to each module method instead of in this passage. I can add a line saying something like each module's methods describe whether consensus or consensus + hotspots were used. (We are also using in the oncoprints, and may need to update co-occurrence).

Reply:
>The other thing that came up as I was reading was that we have not been consistent in active vs. passive voice. Correcting this overall will likely require a more comprehensive review of the manuscript, but we should probably make a decision sooner rather than later to save on future rewrites.

I noticed this - active?

Reply:
I will put in a separate ticket for changing the voice